### PR TITLE
Introduce framework for smoke tests

### PR DESCRIPTION
**Pull request by mmichal10**
This change is [Reviewable](https://reviewable.io/reviews/open-cas/open-cas-linux/9)

### CODE REVIEW

File: test/smoke_test/README [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Both of these have a nice help and detailed output so you should have no
+problems testing.
+
+******************************************************************************
+
+2) Framework files
+
+cas_config - holds default settings for the framework like default cache/core device, mountpoint etc.
+cas_local_config - your local config file, it isn't included by the repository so that you can override the default settings there
```
Comment (micrakow) [inline test/smoke_test/README]: included *in* the repo

**Comment by KamilLepek** [comment]
retest this please

**Comment by KamilLepek** [comment]
retest this please

**Comment by KamilLepek** [comment]
retest this please

**Pull request merged**